### PR TITLE
#20 Added limited jumping functionality

### PR DESCRIPTION
**Pull request by lmschmid**
Needs to be improved on:
- coins rendered above player jumping
- lack of increasing/ decreasing shadow to give illusion of jumping

Needs to be added:
- Visual cooldown bar to see when you are able to jump again

### CODE REVIEW

File: src/Constants.java [hunk 1]
```diff
@@ -13,4 +13,6 @@
     public static final int     STADIUM_MARGIN_RIGHT = 695;
     public static final double  GUARD_SPAWN_RATE = .98;
     public static final int     CHARACTER_VELOCITY = 550;
+	public static final int JUMP_TIME = 1000;
```
Comment (ms2300) [inline src/Constants.java]: Identation

File: src/Streaker.java [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import javafx.scene.image.Image;
-
+import javafx.geometry.Rectangle2D;
 import java.util.ArrayList;
 
 public class Streaker extends AnimatedImage {
 
+	boolean isJumping;
+	boolean canJump;
```
Comment (ms2300) [inline src/Streaker.java]: Indentation

File: src/Streaker.java [hunk 3]
```diff
@@ -13,6 +16,8 @@ public Streaker() {
         this.setFrame(imageArray);
         this.duration = Constants.FRAME_DURATION;
         this.setPosition((Constants.SCREEN_WIDTH / 2) - 40, Constants.SCREEN_HEIGHT / 2);
+		this.isJumping = false;
```
Comment (ms2300) [inline src/Streaker.java]: Indentation

File: src/Streaker.java [hunk 4]
```diff
@@ -47,4 +52,45 @@ public void handleVelocity(ArrayList<String> input) {
             this.addVelocity(0, Constants.CHARACTER_VELOCITY);
         }
     }
+
```
Comment (ms2300) [inline src/Streaker.java:55]: Everything needs to be put onto correct level of rest of the codebase

**Review by ms2300 — CHANGES_REQUESTED** [review]
Can you fix the indentation issues across all the changes to be consistent with rest of the code?

File: src/Constants.java (hunk 1 above)
Comment (lmschmid) [reply src/Constants.java]: This is the only indentation issue that appears on my local level in vim. I'm not sure what you want changed about the rest of them.

Comment (ms2300) [reply src/Constants.java]: I didn't change any code. All of the code you are trying to check is indented incorrectly; I don't know how it looks on your end but it should be changed so it's right on github

**Comment by ms2300** [comment]
Indentation is still not fixed :\

File: src/Streaker.java (hunk 4 above)
Comment (lmschmid) [reply src/Streaker.java:55]: I'm not sure why this is still being flagged, it is a blank line separating two functions.

**Pull request merged**